src/components/AssignmentTexter/StyleControls.js

```diff
@@ -169,19 +183,19 @@ export const flexStyles = StyleSheet.create({
     marginBottom: "8px"
   },
   subSectionMessageFieldTextField: {
-    "@media(max-width: 350px)": {
+    "@media(max-width: 420px)": {
       overflowY: "scroll !important"
     }
   },
   /// * Section Reply/Exit Buttons
   sectionButtons: {
     // TODO: maybe make this contingent on whether there are answer buttons
-    "@media(max-height: 600px)": {
-      flexBasis: "96px" // TODO
-    },
-    "@media(min-height: 600px)": {
-      flexBasis: "144px" // TODO
-    },
+    // "@media(max-height: 600px)": {
```

Comment:
Is there a reason this block of code is commented out instead of deleted?

Reply:
This block of code was impacting my styling; however, since I didn't write this code I wanted to touch base with the team and see if it's ok to delete it.

Reply:
Yup! It's fine to delete